pages/bad-page.js

```diff
@@ -12,49 +11,45 @@ const betterButtons = [
 ];
 
 export default function BadPage() {
-  const router = useRouter();
-  const [buttons, setButtons] = useState(betterButtons);
+  const [aspects, setAspects] = useState(initialAspects);
+  const activeAspects = aspects.filter((aspect) => aspect.status).length;
 
-  function handleSubmit() {
-    window.localStorage.setItems(`buttons`, JSON.stringify(buttons));
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    console.log(data);
   }
 
-  function handleButtonStatus(id) {
-    setButtons(
-      buttons.map((button) => {
-        if (button.id === id) {
-          console.log(button);
-          return { ...button, status: !button.status };
-        }
-        return button;
-      })
+  function handleToggleAspects(id) {
+    setAspects(
+      aspects.map((aspect_) =>
+        aspect_.id === id ? { ...aspect_, status: !aspect_.status } : aspect_
+      )
     );
   }
 
   return (
     <>
-      <h1>What went wrong? </h1>
+      <form onSubmit={handleSubmit}>
+        {aspects.map((aspect) => (
+          <div key={aspect.id}>
+            <label htmlFor={`${aspect}-checkbox`}>{aspect.name}</label>
+            <input
+              name={aspect.id}
+              type="checkbox"
+              id={`${aspect}-checkbox`}
+              checked={aspect.status}
+              onChange={() => handleToggleAspects(aspect.id)}
+            />
+          </div>
+        ))}
 
-      {buttons.map((button) => {
-        return (
-          <button
-            key={button.id}
-            onClick={() => {
-              handleButtonStatus(button.id);
-            }}
-            style={{ backgroundColor: button.status ? "green" : "white" }}
-          >
-            {button.name}
-          </button>
-        );
-      })}
-
-      <Link href="/">
-        <button type="button">Go Back to Menu</button>
-      </Link>
-      <Link href={"/thanks"}>
-        <button onClick={handleSubmit}>Submit</button>
-      </Link>
+        <Link href="/thanks">
+          <button type="submit">Submit</button>
+        </Link>
```

Comment:
```suggestion
          <button type="submit">Submit</button>
```

You should only have a submit button in your form and when submitting your form, your `handleSubmit` should handle the routing.